error: implicit declaration of function 'atoi' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
--- acinclude.m4.orig	2021-10-18 09:20:03.000000000 -0500
+++ acinclude.m4	2021-10-28 02:23:17.000000000 -0500
@@ -38,7 +38,8 @@
 AC_CHECK_DECLS([daylight], , , [#include <time.h>])
 AC_CACHE_CHECK(for daylight, ac_cv_var_daylight,
 [AC_LINK_IFELSE(
-[AC_LANG_PROGRAM([#include <time.h>],
+[AC_LANG_PROGRAM([#include <stdlib.h>
+#include <time.h>],
 [#if !HAVE_DECL_DAYLIGHT
 extern int daylight;
 #endif
